server/models/ClubOverviewModel.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
```

Comment:
this PR and Sad's PR #63 are dead locking (waiting for each other); please consider removing changes related to that PR from this PR

Reply:
I've removed the overlapping changes